antlr4-maven-plugin/src/main/java/org/antlr/mojo/antlr4/Antlr4Mojo.java

```diff
@@ -234,6 +235,8 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
         Log log = getLog();
 
+		outputEncoding = validateEncoding(outputEncoding);
```

Comment:
Fixes a possible bug when no project encoding is provided, introduced with https://github.com/antlr/antlr4/commit/34f0c66a89abfa6d843c34d2f5ed9478a0ed7ef5.

I did not bother to submit a new PR for this.